homeassistant/components/switchbee/const.py

```diff
@@ -0,0 +1,14 @@
+"""Constants for the SwitchBee Smart Home integration."""
+
+from switchbee.device import DeviceType
+
+DOMAIN = "switchbee"
+SCAN_INTERVAL_SEC = 5
+CONF_SCAN_INTERVAL = "scan_interval"
```

Comment:
```suggestion
```
Remove leftover

Reply:
This comment was marked as resolved, but actually not resolved.